ccd-definition/FixedLists/ScannedDocumentType.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ScannedDocumentType",
+    "ListElementCode": "cherished",
+    "ListElement": "Cherished"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ScannedDocumentType",
+    "ListElementCode": "coversheet",
+    "ListElement": "Coversheet"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ScannedDocumentType",
+    "ListElementCode": "form",
```

Comment:
not sure `form` is part of phase 1 as it contains ocr data. and that is phase 2. not a breaking change entry though. just fyi